docs/README.md

```diff
@@ -2,6 +2,51 @@
 title: Cozy-Stack documentation - Table of contents
 ---
 
+Cozy Cloud
+==========
+
+[![go.dev reference](https://pkg.go.dev/badge/github.com/cozy/cozy-stack)](https://pkg.go.dev/github.com/cozy/cozy-stack)
+[![Build Status](https://github.com/cozy/cozy-stack/workflows/CI/badge.svg)](https://github.com/cozy/cozy-stack/actions)
+[![Go Report Card](https://goreportcard.com/badge/github.com/cozy/cozy-stack)](https://goreportcard.com/report/github.com/cozy/cozy-stack)
+
+
+## What is Cozy?
+
+![Cozy Logo](https://cdn.rawgit.com/cozy/cozy-guidelines/master/templates/cozy_logo_small.svg)
+
+[Cozy](https://cozy.io) is a platform that brings all your web services in the
+same private space. With it, your web apps and your devices can share data
+easily, providing you with a new experience. You can install Cozy on your own
+hardware where no one profiles you.
+
+
+## What is the Cozy-Stack
+
+It is the core server of the Cozy platform. It consists of a single process, *the Cozy stack*. 
+
+[Full Cozy-Stack documentation here](https://docs.cozy.io/en/cozy-stack/).
+
+The Cozy-Stack is in charge of serving the Web applications users have installed from the application store.
```

Comment:
I don't think we should talk about the application store just yet. If we talk about it, let's add a link.